src/common/constant.ts

```diff
@@ -226,23 +226,27 @@ export const QUICKSIGHT_USER_NAME_PATTERN =
 export const QUICKSIGHT_NAMESPACE_PATTERN = '^([A-Za-z])[A-Za-z0-9]{4,63}$';
 export const REDSHIFT_DB_NAME_PATTERN = `^${PROJECT_ID_PATTERN}$`;
 export const REDSHIFT_DB_USER_NAME_PATTERN = '^([a-zA-Z][a-zA-Z0-9-_]{1,63})?$';
-export const REDSHIFT_CLUSTER_IDENTIFIER_PATTERN = '^([a-zA-Z][a-zA-Z0-9-_]{1,63})?$';
+export const REDSHIFT_CLUSTER_IDENTIFIER_PATTERN =
```

Comment:
Are it and below lines caused by a format setting or an IDE plug-in?

Reply:
yarn eslint